src/client/views/version/versionDownloadDialog.ml

```diff
@@ -0,0 +1,74 @@
+open Nes
+open Dancelor_common
+open Dancelor_client_model
+open Dancelor_client_html
+open Dancelor_client_components
+open Js_of_ocaml
+
+type t =
+  {
+    choice_rows : Html_types.tr elt list;
+    parameters_signal : VersionParameters.t option React.signal;
+  }
+
+let create () =
+  let (key_choices, key_choices_signal) =
+    Choices.(make [
+        choice [txt "C"] ~checked:true;
+
+        choice [txt "B♭"]
+          ~value:(VersionParameters.make_instrument (Music.make_pitch B Flat (-1)));
+
+        choice [txt "E♭"]
+          ~value:(VersionParameters.make_instrument (Music.make_pitch E Flat 0));
+      ])
+  in
+
+  let (clef_choices, clef_choices_signal) =
+    Choices.(make [
+        choice [txt "𝄞"] ~checked:true;
+
+        choice [txt "𝄢"]
+          ~value:(VersionParameters.make ~clef:Music.Bass ~transposition:(Relative(Music.pitch_c, Music.make_pitch C Natural (-1))) ());
```

Comment:
Nit: Since the key and clef are now on different lines, it seems to suggest that you could have, e.g., a Bass clef with an Bb key. As far as I understand, this will be the case here thanks to the compose function in Transpose, but the transposition argument for the Bass clef seems redundant?

Reply:
The transposition for the bass clef is somewhat necessary to avoid having notes be way too high, it only reduces by one octave. If used in conjunction with a Bb key, the set will have both an octave transposition down (for the bass clef) and a tone transposition down (for the Bb key), effectively being transposed 9 tones down. I'm not exactly sure it answers your question; if it doesn't, then I didn't actually understand the question!

Reply:
Got it, I didn't understand why the transposition for the bass clef was needed. Thanks!